WebPhapp/backend/server.js

```diff
@@ -178,11 +281,6 @@ app.get('/api/v1/patients/:patientID', (req,res) => {
     res.json(matchingPatient);
 });
 
-// get the index
```

Comment:
How come this was removed? Just curious about it.

Reply:
The front end does not route to this API call. If in the future we need Express to provide a specific page we can add that, but we currently don't have a use.

Reply:
Okay, thanks for the clarification.